docs/src/modules/ROOT/pages/frequently-asked-questions.adoc

```diff
@@ -14,6 +14,13 @@ that offers xref:enterprise-edition/enterprise-edition.adoc#enterpriseEditionFea
 to scale out to very large datasets.
 To find out more, see xref:enterprise-edition/enterprise-edition.adoc[Enterprise Edition section] of this documentation.
 
+== Does Timefold offer pre-built models to speed up development?
+
+Timefold offers a suite of pre-built PlanningAI models designed to expedite development by addressing complex scheduling and routing challenges across various industries.
+These models are built upon Timefold Enterprise Solver technology and are accessible through a REST API, facilitating seamless integration into your applications.
```

Comment:
Throughout the docs, we say "Timefold Solver Enterprise Edition". It's a mouthful, I know - but I guess consistency wins?